### PR TITLE
IGNITE-18049 Cursor#close() should not declare any exceptions in throws clause

### PR DESCRIPTION
**Pull request by rpuch**
https://issues.apache.org/jira/browse/IGNITE-18049

### CODE REVIEW

File: modules/rest/openapi/openapi.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@ info:
   license:
     name: Apache 2.0
     url: https://ignite.apache.org
-  version: 3.0.0-SNAPSHOT
+  version: 3.0.0-alpha
```
Comment (ibessonov) [inline modules/rest/openapi/openapi.yaml:9]: What's this?

File: modules/core/src/test/java/org/apache/ignite/internal/util/CursorTest.java [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        cursor.close();
+
+        verify(iterator).close();
+    }
+
+    private interface CloseableIterator extends Iterator<Integer>, AutoCloseable {
+        @Override
+        void close();
+    }
+}
```
Comment (ibessonov) [inline modules/core/src/test/java/org/apache/ignite/internal/util/CursorTest.java:76]: Please add empty line at the end

**Review by ibessonov — APPROVED** [review]

File: modules/rest/openapi/openapi.yaml (hunk 1 above)
Comment (rpuch) [reply modules/rest/openapi/openapi.yaml:9]: Oh... this is an auto-generated file, IIRC, we must commit as soon as it generates with a new version. I committed it accidentally, but it probably needs to be committed. I will ask the guys in Slack.

Comment (rpuch) [reply modules/rest/openapi/openapi.yaml]: Reverted thechange

**Pull request merged**